packages/block-editor/src/components/block-controls/README.md

```diff
@@ -0,0 +1,89 @@
+# BlockControls
+
+When the user selects a particular block, a toolbar positioned above the selected block displays a set of control buttons. Certain block-level controls are automatically included in the toolbar under specific circumstances. For example, there is a control for converting the block into a different type or when the focused element is a RichText component.
```

Comment:
> a toolbar positioned above the selected block

It can optionally be displayed in the top toolbar of the editor. Dunno if this is worth mentioning. 🤷‍♂️

Reply:
Yeah, well I guess it's not a huge miss. As I mentioned, this is a first pass and it can definitely be expanded further to contain the `Slot` and `Fill` usages and maybe a few more examples, but I've tried to make a compromise and have something useful with a few additional links to start with. It's better than nothing 😉